### ISSUE TITLE
genericndkbuild receipe Not compiling with android api > 28

### ISSUE THREAD

**Issue by kengoon** [kivy/python-for-android#2555]
Whenever I am compiling my apk with the `webview` bootstrap:

Buildozer compilation crashes with the below error

```
Android NDK: Found platform level in /home/kengo/.local/share/python-for-android/build/bootstrap_builds/webview/project.properties. Setting APP_PLATFORM to android-30.    
Android NDK: android-30 is above the maximum supported version android-29. Choose a supported API level or set APP_PLATFORM to "latest".    
/home/kengo/.buildozer/android/platform/android-ndk-r20b/build/core/setup-app-platform.mk:101: *** Android NDK: Aborting.    .  Stop.
```

The only solution, I have as of now is to download android ndk 21d to be able to support android API 29 to 30, which works well and also if I want to target android API 31 I will first compile with android API 28 or 29 or 30 depending on the version of ndk, then later on compile again with android API 31

Also it seems that webview bootstrap has been ignored for sometime, at least some update will help

**Comment by misl6** [comment]
Could potentially be fixed via #2550 

Did you had a chance to test?

**Comment by kengoon** [comment]
> Could potentially be fixed via #2550 
> 
> Did you had a chance to test?

Ya

**Comment by kengoon** [comment]
I can confirm that this is solved via #2550

**Comment by misl6** [comment]
Fixed via #2550
